deployment/docker/testnet/node_init.sh

```diff
@@ -69,6 +69,7 @@ then
   secretd q register secret-network-params 2> /dev/null
 
   secretd configure-secret node-master-key.txt "$SEED"
+  secretd configure-old-secret node-master-cert.der "$SEED"
```

Comment:
`configue-old-secret` is not a very intuitive name. Maybe configure-secret-legacy (or even a flag on the same configure-secret command?)